config/introspection.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+type Trace struct {
+	Name        string   `yaml:"name" json:"name"`
+	Probability *float64 `yaml:"probability" json:"probability"`
+	Jaeger      Jaeger   `yaml:"jaeger" json:"jaeger"`
+}
+
+type Jaeger struct {
+	Agent struct {
+		Endpoint string `yaml:"agent_endpoint" json:"endpoint"`
+	}
+	Collector struct {
+		Endpoint string  `yaml:"collector_endpoint" json:"endpoint"`
+		Username *string `yaml:"username" json:"username"`
+		Password *string `yaml:"password" json:"password"`
+	}
+	ServiceName string            `yaml:"service_name" json:"service_name"`
+	Tags        map[string]string `yaml:"tags" json:"tags"`
+	BufferMax   int               `yaml:"buffer_max" json:"buffer_max"`
+}
```

Comment:
This changes the config by removing the `inline` annotation to the struct tags.

The newly un-inlined keys are now `Capped` instead of `lower`.

The configuration options now stutter: `Agent: {agent_endpoint: ""}`.

These changes shouldn't be made in this commit, if at all.

Reply:
A subset of these changes are already in master, these changes came in to remedy a rebase.